documentation/src/main/wiki/AETIn10Minutes.md

```diff
@@ -8,19 +8,19 @@ Before start make sure that you have enough memory on your machine (8 GB is mini
 You need to download and install the following software:
    * [VirtualBox 5.1.30](https://www.virtualbox.org/wiki/Download_Old_Builds_5_1)
    * [Vagrant 1.9.2](https://releases.hashicorp.com/vagrant/)
-   * [ChefDK 1.1.16](https://downloads.chef.io/chef-dk/)
+   * [ChefDK 2.4.17](https://downloads.chef.io/chef-dk/)
    * [Maven](https://maven.apache.org/download.cgi) (at least version 3.0.4)
    * [JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
    * [Chrome browser](https://www.google.com/chrome/browser/desktop/) to view reports
 
 ### Set Up Vagrant
 
-Open a command prompt as the administrator and execute the following commands:
+Open a command prompt **as the administrator** and execute the following commands:
 * `vagrant plugin install vagrant-omnibus`
 * `vagrant plugin install vagrant-berkshelf`
 * `vagrant plugin install vagrant-hostmanager`
 
-Navigate to the `vagrant` module directory. Run `berks install` and then `vagrant up` to start the virtual machine. This process may take a few minutes.
+Clone the AET source code and navigate to the `vagrant` module directory. Run `berks install` and then `vagrant up` to start the virtual machine. This process may take a few minutes.
```

Comment:
Maybe good idea here is to provide the link or command?
```
git clone git@github.com:Cognifide/aet.git
```
There is also the alternative to cloning for people who don't have GitHub account - download zipped repo archive.

Reply:
done